app/src/main/java/com/scouts/kitchenplaner/datalayer/daos/RecipeDAO.kt

```diff
@@ -58,19 +64,76 @@ interface RecipeDAO {
         return recipeId
     }
 
+    @Query("UPDATE recipeEntity SET title = :newTitle WHERE id = :id")
+    suspend fun updateRecipeName(id: Long, newTitle: String)
+
+    @Query("UPDATE recipeEntity SET description = :newDesc WHERE id = :id")
+    suspend fun updateRecipeDescription(id: Long, newDesc: String)
+
```

Comment:
Why is it not also a Query function but an update function (or vice versa?)

Reply:
```@Query``` annotations can take any SQLite query as their parameter (i.e. you could also do Deletes and Inserts with them), the other annotations are essentially just Quality of Life stuff. For small partial updates (in this case just setting a string to a new value if the primary key matches) using an ```@Update``` annotation would mean having to define a DTO just for setting the description. While I do like having many classes this did seem a little excessive.